js/cedar.handlebars.js

```diff
@@ -63,7 +63,7 @@ Handlebars.registerHelper('cedar', function(options) {
 
   var output = '';
 
-  new Cedar.ContentEntry({ cedarId: options.hash.id }).retrieve().then(function(contentEntry){
+  new Cedar.ContentEntry({ cedarId: options.hash.id }).load().then(function(contentEntry){
```

Comment:
Changed the name of this because I wanted to differentiate it from functions that hit the remote server. Calling `load` now MIGHT trigger a server request, but the content will always be `load`ed from `localStorage`, never from the server directly.